src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/factory/events/cas2/Cas2StatusFactory.kt

```diff
@@ -3,11 +3,17 @@ package uk.gov.justice.digital.hmpps.approvedpremisesapi.factory.events.cas2
 import io.github.bluegroundltd.kfactory.Factory
 import io.github.bluegroundltd.kfactory.Yielded
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.events.cas2.model.Cas2Status
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.events.cas2.model.StatusDetailItem
 
 class Cas2StatusFactory : Factory<Cas2Status> {
   private var name: Yielded<String> = { "moreInfoRequested" }
   private var label: Yielded<String> = { "More information requested" }
   private var description: Yielded<String> = { "More information about the application has been requested" }
+  private var statusDetails = listOf(
```

Comment:
I think it might be worth adding a bit more specificity to the commit message for this change to "Added status details to persisted domain event **factory object** so that application status update GET test ensures they are returned..." as I was initially expecting this commit to update the real persisted object (i.e. non-test data).

Reply:
Done this - thanks George.